### DISCUSSION TITLE
Assigning Reviews

### DISCUSSION THREAD

**Discussion by SudharakaP** [opencollective/opencollective#3972] [Development Q&A] [answered]
@opencollective/core : I had this question for a while but didn't quite get the chance to ask. 

So when a PR is completed what's the best way to assign it for review. Should we assign it to @opencollective/core or an individual developer? Is there some kinda consensus of this? 😃

**Comment by Betree** [comment, marked as answer]
- For the core team, we assign PRs to each other based on who are the project owners (we normally publish that info in sprint plannings on Github, but it looks like the last one hasn't been published). Each project has at least two technical owners. It's also fine to include other team members if we want to.

- For contributors, it's fine to assign to the developer who created the spec/bounty for the issue if there's one. Otherwise, you can indeed assign it to @opencollective/core and we will dispatch. We usually look at unassigned contributors PRs once a week, so if you assign nobody it will eventually get picked up anyway. If a quick review is expected, it's also a good practice to share your PR in Slack's #engineering channel.

**Reply by SudharakaP** [reply]
Thanks much for the detailed explanation. That clarifies it. 😄
